documentation/asciidoc/accessories/camera/libcamera_apps_multicam.adoc

```diff
@@ -0,0 +1,12 @@
+=== Multiple Cameras Usage
+
+Basic support for multiple cameras is available within `libcamera-apps`. Multiple cameras may be attached to a Raspberry Pi in the following ways:
+
+* Two cameras connected directly to a Raspberry Pi Compute Module board, see link:https://www.raspberrypi.com/documentation/computers/compute-module.html#attaching-a-raspberry-pi-camera-module[here] for more details.
```

Comment:
I guess it doesn't really matter, but in general we've not been using the [link macro](https://docs.asciidoctor.org/asciidoc/latest/macros/link-macro/) and have just been doing e.g. `https://www.raspberrypi.com/documentation/computers/compute-module.html#attaching-a-raspberry-pi-camera-module[here]`. And IIRC @aallan is a bit averse to "see here" links on titles?

Oh! And this is a relative-link, so it ought to be a relative [xref](https://docs.asciidoctor.org/asciidoc/latest/macros/inter-document-xref/) rather than an explicit "external" link to `https://www.raspberrypi.com/documentation/...`

Reply:
Sure, will fix these.